public/css/mixin/progress-bar.less

```diff
@@ -9,23 +9,32 @@
 
     .above {
       padding: ~"calc(1em + 1px) 10%";
+
+      .now {
+        width: auto;
+        z-index: 2;
+      }
     }
 
     .below {
       padding: 1.25em 10%;
     }
 
+    .placement {
+      position: absolute;
+    }
+
     .bubble {
       .rounded-corners(.25em);
       background-color: @body-bg-color;
       border: 1px solid @gray-light;
-      position: absolute;
+      position: relative;
       bottom: 0;
```

Comment:
`position` is not required or is it? Didn't made a difference for me when disabled at least.
`bottom` however is definitely useless for relative positioning anyway.

Reply:
`.bubble::hover{z-index}` has no effect when the position is removed.